cmd/common.go

```diff
@@ -137,11 +141,24 @@ func syncMain(ctx context.Context, filenames []string, dry bool, parallelism,
 			return fmt.Errorf("reading Kong version: %w", err)
 		}
 	}
-	parsedKongVersion, err = parseKongVersion(kongVersion)
+	parsedKongVersion, err = utils.ParseKongVersion(kongVersion)
 	if err != nil {
 		return fmt.Errorf("parsing Kong version: %w", err)
 	}
 
+	if mode == modeKong || mode == modeKongEnterprise {
```

Comment:
We should default to 3.0 since that is the version in Konnect. Please add a backlog item on the Konnect issue tracker to expose the current control-plane version if it is not already done so.